### PR TITLE
FIX: sync es_419, nl_nl, sl_SI

### PR DESCRIPTION
**Pull request by limpbrains**

### CODE REVIEW

File: loc/nl_nl.json [hunk 1]
```diff
@@ -277,7 +287,7 @@
     "network_electrum": "Electrum server",
     "not_a_valid_uri": "Ongeldige URI",
     "notifications": "Meldingen",
-    "open_link_in_explorer": "Open link in verkenner",
+    "open_link_in_explorer" : "Open link in verkenner",
     "password": "Wachtwoord",
```
[the remark this answers is not in the document]
Comment (Overtorment) [reply loc/nl_nl.json]: nl language is a high entropy string. noted.

[the remark this answers is not in the document]
Comment (limpbrains) [reply loc/nl_nl.json]: Mmmm, entropy...

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by Overtorment** [comment]
poured fresh master

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ahtqs7rqad7ch622ty5l2jktmq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10837)

**Pull request merged**